backend/app/services/jobs.py

```
@@ -0,0 +1,194 @@
+                try:
+                    logger.info("[job] Starting repository cache update task")
+                    
+                    db = SessionLocal()
+                    try:
+                        # Run the repository update job asynchronously
+                        loop.run_until_complete(repository_job_service.update_repositories_for_all_users(db))
+                    finally:
+                        db.close()
+                        
+                    logger.info("[job] Repository cache update task execution completed")
+                except Exception as e:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)